src/Components/ResourceList/Card/ResourceCardEditableDescription.tsx

```diff
@@ -1,150 +1,146 @@
 // Libraries
-import React, {Component, KeyboardEvent, ChangeEvent} from 'react'
+import React, {
+  forwardRef,
+  KeyboardEvent,
+  ChangeEvent,
+  RefObject,
+  useState,
+} from 'react'
 import classnames from 'classnames'
 
 // Components
-import {Input} from '../../Inputs/Input'
+import {Input, InputRef} from '../../Inputs/Input'
 import {Icon} from '../../Icon/Icon'
 import {ClickOutside} from '../../ClickOutside/ClickOutside'
 
 // Types
-import {StandardClassProps, ComponentSize, IconFont} from '../../../Types'
+import {StandardFunctionProps, ComponentSize, IconFont} from '../../../Types'
 
 // Styles
 import './ResourceCardDescription.scss'
 
-interface Props extends StandardClassProps {
+export interface ResourceCardEditableDescriptionProps
+  extends StandardFunctionProps {
+  /** Called when user hits enter or blurs the input  */
+  onUpdate: (description: string) => void
   /** Text to display in description */
   description: string
   /** Placeholder text to display in input during editing */
-  placeholder: string
-  /** Called when user hits enter or blurs the input  */
-  onUpdate: (description: string) => void
+  placeholder?: string
+  /** Pass through to assign a ref to the Input present in edit mode */
+  inputRef?: RefObject<InputRef>
```

Comment:
Nice to have but unnecessary since we have the ref on the parent element.

Reply:
Yeah wasn't sure how useful it is since the input is only visible in edit mode